packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java

```diff
@@ -144,7 +137,6 @@ private TextAttributeProps() {}
   public static TextAttributeProps fromMapBuffer(MapBuffer props) {
     TextAttributeProps result = new TextAttributeProps();
 
-    // TODO T83483191: Review constants that are not being set!
```

Comment:
We're deserializing text attributes of a _fragment_. Some attributes are not handled on the fragment level but apply to the whole `Text` element. I believe there's not much more to investigate here.

Reply:
Relevant:

https://github.com/facebook/react-native/blob/27f38102973849f962801bbb184d98387f7f6022/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java#L31-L33

Reply:
The fact that we currently don't do what this comment describes is blocking me.

I'd like to...
- Split `TextAttributes` (C++) to `TextAttributes` (`<Text>`-level properties)  and `FragmentAttributes` (fragment-level properties) 
- Refactor `TextAttributeProps` (Java) to `FragmentAttributes`

Currently, one is a subset of another; not modeling them separately is at most a code smell.

I'd like them to be two slightly different sets (sharing a great common subset), so not modeling them separately is a problem.

Reply:
yes, this doesn't seem correct, thanks for cleaning up